src/plugins/history/galleria.history.js

```diff
@@ -32,8 +32,8 @@ Galleria.History = (function() {
         get = function( winloc ) {
             if( iframe && !support && Galleria.IE ) {
                 winloc = winloc || iframe.location;
-            }  else {
-                winloc = loc;
+            } else {
+                iframe.location;
```

Comment:
@kennly this isn't doing anything. You could remove the whole `} else {` block to achieve the same.